gengokit/httptransport/httptransport.go

```diff
@@ -101,6 +111,24 @@ func NewBinding(i int, meth *deftree.ServiceMethod) *Binding {
 		nField.LowCamelName = LowCamelName(nField.Name)
 
 		nBinding.Fields = append(nBinding.Fields, &nField)
+
+		// Emit warnings for certain cases
+		if nField.IsBaseType == false {
+			wrn := []string{
+				"%s.%s is a custom type '%s', only base types and repeated base ",
```

Comment:
I'd actually collapse it all down like so:
```go
log.Warnf(
    "%s.%s is a custom type %q, only base types and repeated base " +
        "types are supported. As a result, the generated HTTP " +
        "transport will fail to compile. Remove non-base types.",
    meth.GetName(),
    nField.Name,
    nField.ProtobufType)
```
There's no need to declare a variable for it if all we're doing is using it the log statement below.